go.mod

```diff
@@ -3,22 +3,18 @@ module github.com/redhatinsights/payload-tracker-go
 go 1.14
 
 require (
-	github.com/atombender/go-jsonschema v0.9.0 // indirect
 	github.com/aws/aws-sdk-go v1.40.13
 	github.com/confluentinc/confluent-kafka-go v1.7.0
 	github.com/go-chi/chi/v5 v5.0.3
 	github.com/google/uuid v1.3.0
+	github.com/kr/pretty v0.2.1 // indirect
 	github.com/onsi/ginkgo v1.16.4
 	github.com/onsi/gomega v1.16.0
 	github.com/prometheus/client_golang v1.11.0
-	github.com/qri-io/jsonschema v0.2.1
 	github.com/redhatinsights/app-common-go v1.5.1
 	github.com/redhatinsights/platform-go-middlewares v0.9.0
-	github.com/redis-go/redcon v0.2.0 // indirect
-	github.com/redis-go/redis v0.0.0-20190129163758-e7bb29b08926
```

Comment:
NICE!  I really like the `go mod tidy`!
I have to remember to use that whenever I push code up!

Reply:
Yes, it cleans up unused libraries :100:

Reply:
Yes, it cleans up unused libraries :100: